strawberryfields/gbs/pp.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Point Processes
+===============
+**Module name:** :mod:`strawberryfields.gbs.pp`
+
+.. currentmodule:: strawberryfields.gbs.pp
+
+This module contains functions for point process utilization.
+
+Summary
+-------
+
+.. autosummary::
+    rbskernel
+
+Code details
+^^^^^^^^^^^^
+"""
+
+import numpy as np
+from scipy.spatial.distance import cdist
+
+
+def rbfkernel(R, sigma):
```

Comment:
since we are only offering one type of kernel, how about just calling this `kernel`?
This is also the "canonical" kernel (?), so I think it's fair to just call this kernel.

We also use type hinting in the apps layer:
```suggestion
def rbfkernel(R: np.ndarray, sigma: float) -> np.ndarray:
```